### PR TITLE
Fixing Edge Definition Inconsistency

### PR DESCRIPTION
**Pull request by MikeGost**
Most important change here: From a previous PR (https://github.com/osmlab/atlas/pull/45) - tag criteria for what becomes an `Edge` when converting from PBF was updated. However, when building relations, we were looking at the Edge definition as provided by the `atlas-edge.json` file. Essentially - two definitions of `Edge` (see `TagMap` > `matchAtlasEdge` > `PbfOneWay` and the configuration file) This is an inconsistency that will be addressed by Part 3 of the PBF Ingest Refactor (see Raw Atlas PRs). However, in the meantime, the fix is to bring these two definitions in sync. This PR does just that and creates a test for `Relation` [4451979](https://www.openstreetmap.org/relation/4451979), which exhibits some of the complex tagging that was affected. 

Other changes include:
1. Cleaning up logging in `CountryBoundaryMap`. This has the potential to flood Spark logs with information about the time it took to slice each Way. Most slice operations take tens or at most hundreds of milliseconds and the logs would be overly verbose if we log each and every slice. Reducing the logging to only log in case of abnormal (greater than 1 minute) slicing time. 
2. Adding new `PublicServiceVehiclesTag` - encountered during investigation. It may prove to be relevant for those interested in specific road designation.
3. Fixing integration test that updates the members of a `Relation`. Specifically, Way [83642691](https://www.openstreetmap.org/way/83642691) became a bi-directional Edge instead of a Line. It was way-sectioned 3 times, and before had 3 Lines representing it, whereas now it has 6 edges (2 in each direction) representing it. The atlas excerpt:

Before:
```
[Relation: id=1251624000000, [Members: 
	{Member: ID = 83642691000001, Type = LINE, Role = }, 
	{Member: ID = 83642691000002, Type = LINE, Role = }, 
	{Member: ID = 83642691000003, Type = LINE, Role = }, ...
```

After:
```
[Relation: id=1251624000000, [Members: 
	{Member: ID = -83642691000003, Type = EDGE, Role = }, 
	{Member: ID = -83642691000002, Type = EDGE, Role = }, 
	{Member: ID = -83642691000001, Type = EDGE, Role = }, 
	{Member: ID = 83642691000001, Type = EDGE, Role = }, 
	{Member: ID = 83642691000002, Type = EDGE, Role = }, 
	{Member: ID = 83642691000003, Type = EDGE, Role = }, ...
```

### CODE REVIEW

File: src/main/resources/org/openstreetmap/atlas/geography/atlas/pbf/atlas-edge.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
     "filters": [
-        "access->!no",
+        "access->!no|motor_vehicle->yes|motorcar->yes|vehicle->yes",
```
Comment (mertemin) [inline src/main/resources/org/openstreetmap/atlas/geography/atlas/pbf/atlas-edge.json]: It'd be nice to have a comment in `PbfOneWay`'s referencing this file.

Comment (MikeGost) [reply src/main/resources/org/openstreetmap/atlas/geography/atlas/pbf/atlas-edge.json]: Good call!

File: src/integrationTest/java/org/openstreetmap/atlas/geography/atlas/pbf/OsmPbfLoaderIntegrationTest.java [hunk 2]
```diff
@@ -228,7 +228,7 @@ public void testRoute()
         Assert.assertEquals("route", relation1.getTags().get("type"));
         Assert.assertEquals("bus", relation1.getTags().get("route"));
         Assert.assertEquals("Adult Tram", relation1.getTags().get("name"));
-        Assert.assertEquals(24, relation1.members().size());
+        Assert.assertEquals(27, relation1.members().size());
```
Comment (matthieun) [inline src/integrationTest/java/org/openstreetmap/atlas/geography/atlas/pbf/OsmPbfLoaderIntegrationTest.java]: It is always better to have a test that is not wrong ;)

File: src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java [hunk 3]
```diff
@@ -1027,7 +1028,10 @@ public int size()
             }
         }
 
-        logger.info("Took {} to slice way {}", time.untilNow(), identifier);
+        if (logger.isDebugEnabled() && time.untilNow().isMoreThan(Duration.ONE_MINUTE))
+        {
+            logger.info("Took {} to slice way {}", time.untilNow(), identifier);
```
Comment (matthieun) [inline src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java]: Might want to do `logger.debug` here.

Comment (MikeGost) [reply src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java]: Oops, had this and it changed back to info after fixing the branch, nice catch!

**Comment by MikeGost** [comment]
@mertemin - updated slightly to remove some of the functionality you added in your last [PR](https://github.com/osmlab/atlas/pull/45/files#diff-30b1963a2c456670fed8cf1fbd1bb370R80). Take a look at this one more time please. cc. @matthieun for updates per our last chat.

**Pull request merged**